kunstmaan/menu-bundle/5.2/config/packages/kunstmaan_menu.yaml

```diff
@@ -0,0 +1,6 @@
+#kunstmaan_menu:
+#    menus:
+#        - main_navigation
```

Comment:
I think you should remove or uncomment this

Reply:
This is more to give the user a example config which they can uncomment and change to their needs. The names will show up in the interface, so the menu name will likely be changed by the user. We added this like the liip/imagine-bundle adds an [example config](https://github.com/symfony/recipes-contrib/blob/bce79fc5f07e850c0d53d13fcbd5978d740a6a6f/liip/imagine-bundle/1.8/config/packages/liip_imagine.yaml)